libs/SmartStore/SmartStoreTests/SFSmartStoreFullTextSearchTests.h

```diff
@@ -27,5 +27,12 @@
 
 #import "SFSmartStoreTestCase.h"
 
-@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase 
+@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase
+
+#define kTestStore            @"testSmartStoreFullTextSearchStore"
```

Comment:
@kchitalia ```#define``` is a preprocessor directive, so basically it's a macro - fancy way of doing ```Find and Replace```. Every occurrence of that key is substituted by the value. ```static const``` creates a single copy of that variable, and allocates memory to it, which is common to all instances of that object, so it's always in memory. That's basically the difference.